### ISSUE TITLE
App not update Getting key not possible! The Solution

### ISSUE THREAD

**Issue by SpoturDeal** [nabbl/node-red-contrib-google-home-notify#27]
It seems that this app is no longer maintained.
I have deleted it from my palette.

Use **node-red-contrib-cast**  this one works perfect.

**Comment by nabbl** [comment]
Yes unfortunately I don't have time to maintain this feature at the moment. Additionally I don't even need to use it myself anymore. So the incentive is kind of gone.

I updated the Readme to help users find your alternative library.

**Comment by nabbl** [comment]
Leaving this open for a while.

**Comment by specklesmike** [comment]
Note: 2019
In ~/.node-red/node_modules/google-home-notify`

edit package.json and change the dependency on 'google-tts-api' from '0.0.2-alpha1' to '0.0.3', so it reads:

"dependencies": {
"bluebird": "3.5.1",
"castv2-client": "^1.2.0",
"google-tts-api": "**0.0.4**"
}

Update the dependency
`npm update`

Restart node-red
`node-red-stop`
`node-red-start`
